store/tikv/lock_resolver.go

```diff
@@ -342,7 +350,9 @@ func (lr *LockResolver) resolveLocks(bo *Backoffer, callerStartTS uint64, locks
 				cleanTxns[l.TxnID] = cleanRegions
 			}
 
-			if l.LockType == kvrpcpb.Op_PessimisticLock {
+			if status.useAsyncCommit {
```

Comment:
```suggestion
			if status.useAsyncCommit && ! exists {
```
When meets several locks belonging to an async-commit transaction, we only have to resolve it once.